feat: Index Account Changes

khorolets opened the pull request:
Waiting for https://github.com/near/nearcore/pull/3668 to be merged

Closes #9

frol commented on src/models/state_changes.rs:
```diff
@@ -0,0 +1,83 @@
+    pub affected_account_storage_usage: BigDecimal,
+}
+
+impl StateChange {
+    pub fn from_state_change_with_cause(
```
rename to address "account changes" naming

frol commented on src/models/state_changes.rs:
```diff
@@ -0,0 +1,83 @@
+                BigDecimal::from_str(acc.locked.to_string().as_str())
+                    .expect("`locked` expected to be u128")
+            } else {
+                BigDecimal::from_str("0")
+                    .expect("`locked` expected to be u128")
```
```suggestion
                BigDecimal::from(0)
```

Update the rest

frol commented on src/models/enums.rs:
```diff
@@ -95,3 +96,35 @@ impl From<&near_indexer::near_primitives::account::AccessKeyPermission> for Acce
+            near_indexer::near_primitives::views::StateChangeCauseView::PostponedReceipt { .. } => Self::PostponedReceipt,
+            near_indexer::near_primitives::views::StateChangeCauseView::UpdatedDelayedReceipts { .. } => Self::UpdatedDelayedReceipts,
+            near_indexer::near_primitives::views::StateChangeCauseView::ValidatorAccountsUpdate { .. } => Self::ValidatorAccountsUpdate,
+            StateChangeCauseView::NotWritableToDisk => { panic!("Not expected to get NotWritableToDisk here") }
```
Let's use tagged matching to avoid duplication of panics:

```
unexpected_state_change_cause @ NotWitableToDisk | InitialState => panic!("Unexpected state change cause {:?}", unexpected_state_change_view)
```

khorolets commented:
@frol I've switched order of two last migrations as indices were applied earlier.

We've changed primary key for `account_changes` table from composite to `bigserial` because Postgres forces our nullable fields to be non-nullable implicitly but we can't afford that (see stackoverflow answer for [ref](https://stackoverflow.com/questions/26642835/composite-primary-key-enforces-not-null-constraints-on-involved-columns))
